src/git/util.js

```diff
@@ -23,7 +29,8 @@ const defaultBranchTemplate = '{issue.shortName}/{issue.key}_{issue.sanitizedSum
 
 const getTemplateData = (config, issue) => ({
   template: config.get(['jira', 'templates', 'branch']) || defaultBranchTemplate,
-  data: {
+  matchers: TEMPLATE_KEY_MATCHERS,
```

Comment:
`TEMPLATE_KEY_MATCHERS` does not depend on `config` or `issue`, so it can be directly accessed in `getIssueIdFromBranch`.